developers/wallets/native/index.md

```diff
@@ -119,23 +124,70 @@ The `request` value is URL encoded. Its contents look like this when unencoded:
+Depending on what you declared in your `acceptedProtocols` you may receive either or both as properties in the `protocols` object carried in the `request` query parameter. Your Wallet's code (i.e. in `/switchboard`) should decide which protocol to use (or it may delegate this choice to advanced users) when multiple protocols are available.
 
 ### VC-API
 
-The `vcapi` property will be a URL that will tell the wallet where to go to retrieve the credential. The wallet can send a `POST` request to this URL with an empty JSON object (`{}`) to attempt to download the VC to be stored. The VC will be found in the `verifiablePresentation` object of the response to that `POST` request. If there are further steps required in the flow such as DID Auth a `verifiablePresentationRequest` object will describe the details.
+The `vcapi` property will contain a URL that tells the wallet where it can the credential.
+
+To retrieve the credential via VC-API, the wallet first sends a `POST` request
+to the URL (extracted from `protocols.vcapi` above) with an empty JSON object
+(`{}`) in the body of the request to attempt to download the VC to be stored.
+
+```http
+POST /exchangers/z1A1GqykGBWKbwhFCDqFjMfnG/exchanges/z19mxa763DAKX7diL51kBFecZ
+Host: exchanger.example.com
+Content-Type: application/json
 
-See the [VC API Specification](https://w3c-ccg.github.io/vc-api/) for more details on this protocol.
+{}
+```
+
+The response to that request will contain a Verifiable Credential within a
+`verifiablePresentation` object. Additionally, if there are further steps
+required in the flow such as DID Auth a `verifiablePresentationRequest` object
+will describe the details.
 
-Example VCAPI protocol URL:
-`https://exchanger.example.com/exchangers/z1A1GqykGBWKbwhFCDqFjMfnG/exchanges/z19mxa763DAKX7diL51kBFecZ`
+Below is a partial example response:
+
+```json
+{
+  "verifiablePresentation": {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://www.w3.org/2018/credentials/examples/v1",
+      "https://w3id.org/security/suites/ed25519-2020/v1"
+    ],
+    "type": ["VerifiablePresentation"],
+    "holder": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "verifiableCredential": [{
+          "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://www.w3.org/2018/credentials/examples/v1",
+        "https://w3id.org/security/suites/ed25519-2020/v1"
+      ],
+      "id": "http://example.edu/credentials/3732",
+      "type": [
+        "VerifiableCredential",
+        "UniversityDegreeCredential"
+      ],
+      ...
+    }]
+  }
+}
+```
+
+More complete examples can be found in the
+[Example Exchanges](https://exchanger.example.com/exchangers/z1A1GqykGBWKbwhFCDqFjMfnG/exchanges/z19mxa763DAKX7diL51kBFecZ)
```

Comment:
- Looks like the former example URL.  Was that supposed to be preserved somewhere?
- I assume this should now be a spec anchor link?

Reply:
It'll link to https://w3c-ccg.github.io/vc-api/#exchange-examples in the next update.